lang/en/docs/external/status.md

```diff
@@ -0,0 +1,21 @@
+# External Upload Status Indicators
+
+Items listed under the External Uploads [Explorer](ui/explorer.md) can be in one of the following possible statuses, appearing under its corresponding letter/color badge.
+
+## Active
+
+Badge: <span class="btn badge b-warning border-50">A</span>
+
+The "Active" status highlights the fact that the External Upload task is currently in the process of being executed.
+
+## Finished
+
+Badge: <span class="btn badge b-success border-50">F</span>
+
+When the (upload) task is "Finished", (without errors) the status is updated accordingly.
+
+## Error
+
+Badge: <span class="btn badge b-danger border-50">E</span>
+
+An "Error" status indicates that the task execution terminated as a result of encountering a computational error.
```

Comment:
There is no computation. It is an execution|parsing error.

Reply:
Fixed